### DISCUSSION TITLE
Need help configuring helix for android-development.

### DISCUSSION THREAD

**Discussion by imtheaman** [helix-editor/helix#8384] [Q&A]
have already installed kotlin-lang-server but dont' know how to set it up completely. could anyone help?

**Comment by imtheaman** [comment]
have already checked reddit page. couldn't find anything related out. looks like most people are using helix for either web-dev or backend

**Comment by pyranota** [comment]
@imtheaman Any progress on that? Did you manage to make lsp work?

**Comment by dessalines** [comment]
I'm desperate for a working android dev solution with helix.

**Comment by imtheaman** [comment]
Hey @dessalines , @pyranota!
Sorry for the late reply, wasn’t active in this account. I’ve shifted to nvim for most of the things and have configured android studio as well quite close to my nvim setup, for iOS I’m still doing it in Xcode. It has shitty vim support.

**Comment by imtheaman** [comment]
Btw in vim I had it working once but for running the app and making a debug or release build it was terminal only and I thought android studio is easier so ditched that. There was a tool already built which I had configured along with something else I don’t remember now. It was all pre Ai era

**Comment by dessalines** [comment]
I'd be interested, but I'm mostly interested in helix support, not nvim.

**Comment by imtheaman** [comment]
You can use Claude to build the plugin yourself though if you’re accustomed to helix for day to day development

-------- Original Message --------
